core/Tests/Connection.hs

```diff
@@ -213,6 +215,12 @@ arbitraryRSACredentialWithUsage usageFlags = do
     cert <- arbitraryX509WithKeyAndUsage usageFlags (PubKeyRSA pubKey, ())
     return (CertificateChain [cert], PrivKeyRSA privKey)
 
+readSessionRef :: (IORef client, IORef server) -> IO client
```

Comment:
This function name is confusing to me. I like `readClientSessionRef` or something.

Reply:
Renamed.